src/main.rs

```diff
@@ -159,6 +157,26 @@ fn run_machete() -> anyhow::Result<bool> {
     Ok(has_unused_dependencies)
 }
 
+fn remove_dependencies(manifest: &str, dependencies_list: &[String]) -> anyhow::Result<String> {
+    let mut manifest = toml_edit::Document::from_str(manifest)?;
+    let dependencies = manifest
+        .iter_mut()
+        .filter(|x| x.1.is_table_like())
+        .filter(|(k, _)| k == "dependencies")
+        .next()
+        .context("no dependencies table found")?
+        .1
+        .as_table_mut()
+        .expect("Checked above");
```

Comment:
Any preference to make this an `expect` instead of plain returning an err result with `?` here?